Modules/Microsoft365DSC/DSCResources/MSFT_TeamsUpdateManagementPolicy/readme.md

```diff
@@ -0,0 +1,4 @@
+
+# TeamsUpdateManagementPolicy
+
+This resource configures the Teams Update policies.
```

Comment:
Maybe also add a reference to documentation of this policy: https://docs.microsoft.com/en-us/MicrosoftTeams/public-preview-doc-updates#set-the-update-policy